src/Themes/ThemeManager.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Carnival\Themes;
+
+use Carnival\Themes\Theme;
+use Illuminate\Support\Collection;
+
+class ThemeManager {
+
+    protected Collection $themes;
+
+    public function __construct() {
+        $this->themes = collect();
+    }
+
+    function register(Theme $theme) : void {
+        $this->themes->put(get_class($theme), $theme);
+    }
+
+    function get(string $themeClass) : Theme {
```

Comment:
What happens here if there is no theme under the requested `$themeClass`?
Perhaps this should return an optional?

Reply:
Okay I'll change it to return ?Theme